annotations/Avercage_-_Embers.csv

```diff
@@ -0,0 +1,43 @@
+start_time,end_time,lyrics_line
+32.34,34.08,Through days of thunders

```

Comment:
The timing does not add up with the timings we already have in the `wordonset.txt` files, for this line it should be 32.4483759123 as start and 34.2626158703 as end (start of next word). Or did you annotate the lines manually on your own to get new timestamps?

Reply:
Hi! @f90. Yes..., actually I aligned the time codes manually to get the start and end times for evaluation on end to end transformer models. I could only input segments with limited length up to only 13 sec to the model. Do you have better suggestions? Thanks!

Reply:
Ah I see. So you have reliable timestamps for the end of the words at each of these lyrical lines, right? In that case what about we keep the end timestamps as you manually annotated them, and we use the start timestamps from the start of the first word in each line (which we already have annotated in this dataset) as the start timestamp of the line? 

So that would mean you only need to update the `start_time` for each row that you submitted in this PR (and make sure that this new start time is after the end time of the last line and before the end of the current line, but you can correct the end times in these cases as well if they are the issue). I think then I would be very happy to accept this data